src/components/MenuAdmin.tsx

```diff
@@ -0,0 +1,94 @@
+import React from 'react';
+import styled from '@emotion/styled';
+import { Button, Typography } from '@mui/material';
+import { Link } from 'react-router-dom';
+import StyledLink from './StyledLink';
+
+const MenuAdminContainer = styled.div`
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  justify-content: flex-start;
+  padding: 0px 36px;
+  gap: 16px;
+
+  height: 64px;
+`;
+
+const MenuAdminItem = styled.div`
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  padding: 0px 8px;
+  height: 64px;
+  background: #ffffff;
+`;
+
+const MenuAdminItemTypo = styled(Typography)`
+  min-height: 24px;
+
+  font-family: 'Roboto';
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 150%;
+  letter-spacing: -0.04px;
+
+  color: #6e6d7a;
+`;
+
+const PresentItem = styled(MenuAdminItem)`
+  box-shadow: inset 0px -2px 0px rgba(0, 0, 0, 0.87);
+`;
+
+const PresentItemTypo = styled(MenuAdminItemTypo)`
+  color: rgba(0, 0, 0, 0.87);
+`;
+
+interface MenuAdminProps {
+  adminTab: AdminTabs;
+}
+
+export type AdminTabs = 'injection-point' | 'registration' | 'doccument';
+
+const MenuAdmin = ({ adminTab }: MenuAdminProps) => {
```

Comment:
trông đoạn này code tay to thế? Thế mà có chục cái menu thì copy đi copy lại à e?